rust/processor/src/models/account_transaction_models/account_transactions.rs

```diff
@@ -42,13 +41,19 @@ impl AccountTransaction {
     /// TODO: include table items in the detection path
     pub fn from_transaction(transaction: &Transaction) -> AHashMap<AccountTransactionPK, Self> {
         let txn_version = transaction.version as i64;
-        let txn_data = transaction.txn_data.as_ref().unwrap_or_else(|| {
-            error!(
-                transaction_version = transaction.version,
-                "Txn Data doesn't exist for version {}", transaction.version
-            );
-            panic!();
-        });
+        let txn_data = match transaction.txn_data.as_ref() {
+            Some(data) => data,
+            None => {
+                PROCESSOR_UNKNOWN_TYPE_COUNT
+                    .with_label_values(&["AccountTransaction"])
+                    .inc();
+                tracing::warn!(
```

Comment:
Maybe helpful to keep `transaction_version = transaction.version` for structured log? (same as others below)

Reply:
good call; fixed now.